code/components/citizen-resources-core/src/StateBagComponent.cpp

```diff
@@ -503,7 +503,7 @@ bool StateBagComponentImpl::IsSafePreCreateName(std::string_view id)
 
 	for (const auto& prefix : m_preCreatePrefixes)
 	{
-		if (id.find(prefix) == 0)
+		if (id.rfind(prefix, 0) == 0) // equivalant of a starts with
```

Comment:
`equivalent`, with an `e`.

I think C++20 has an explicit starts-with but we're still not in that compiler mode.